.evergreen/config.yml

```diff
@@ -330,6 +330,26 @@ functions:
           done
 
   "install swift":
+    - command: shell.exec
+      params:
+        working_dir: "src"
+        script: |
+          set -o xtrace
+          set -o errexit
+
+          if [ "${SWIFT_MINOR_VERSION}" = "main-snapshot" ]; then
```

Comment:
what does "main-snapshot" signify here?

Reply:
"main-snapshot" is basically just a placeholder string we use in the Evergreen config to mean "latest development snapshot from the Swift main branch". 
in the Swift installation/configuration scripts, if we see that version, we look at swiftenv's available snapshots to determine the actual version to use, which is usually something like DEVELOPMENT-SNAPSHOT-2022-01-09:

https://github.com/mongodb/mongo-swift-driver/blob/d63c53d4222f0f2277436e77e0d55cac0dec400f/.evergreen/install-swift.sh#L23-L25